unitary/quantum_chess/mcpe_utils_test.py

```diff
@@ -57,7 +57,7 @@ def test_effect_of_swap():
 
 
 def test_distance_fn():
-    a1, a2, a3, b1, b2, b3 = cirq.GridQubit.rect(2, 3)
+    a1, a2, a3, _b1, b2, _b3 = cirq.GridQubit.rect(2, 3)
```

Comment:
what's with underscores on some but not all of them?  Can we make it consistent?0.
same comment in unitary/quantum_chess/swap_updater_test.py

Reply:
These were flagged by the unused-variable pylint checker.
I can change them to `_` if you think it is more readable.

Reply:
Yeah, let's change them all to use underscores, just to be consistent.  Thanks for the cleanup!

Reply:
Sure!  Done in d470f10f23d2e8d8f712cf4dce539c5a8144a3db.